File: Python/level2/예상 대진표.py
def solution(n,a,b):
    answer = 0
    while a != b:
        answer += 1
        a = a / 2 if a % 2 == 0 else a // 2 + 1
        b = b / 2 if b % 2 == 0 else b // 2 + 1
    return answer

File: 예상대진표.cpp
//0210 예상 대진표 by CJHofficial

#include <iostream>

using namespace std;

int solution(int n, int a, int b)
{
    int answer = 1;
    
    while((a/2 == b/2) || (abs(a-b) != 1)){
        a = (a-1)/2 + 1;
        b = (b-1)/2 + 1;
        answer += 1;
    }
    return answer;
}